samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -149,6 +149,13 @@
             "metadata": {
                 "description": "The key for the Text Analytics subscription."
             }
+        },
+         "TextAnalyticsEndpoint": {
```

Comment:
I see that TextAnalyticsRegion is still part of the ARM template - please remove it if it is not used/needed anymore.